CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change log
 
+## main (unreleased)
+
+  * Updated nokogiri dependency from (~> 1.10.1 to ~> 1.11.1)
```

Comment:
I see what you're going for, but could we leave the (lack of) indentation consistent with the bullet from the previous version?

Reply:
I tried that and got an error from the linter, this was my attempt to resolve it. Not sure what it wants 🤔